.github/workflows/getBuildLists.py

```diff
@@ -0,0 +1,23 @@
+import sys
+import json
+
+
+def main():
+    print(sys.argv)
+    result = []
+    for argument in sys.argv:
+        if (argument.startswith('perf/')):
```

Comment:
Not sure if the parse take consideration of duplication?

Reply:
Yes, I didn't realize it! 
Fixed it by adding an additional condition check.